ideas/README.md

```diff
@@ -1,8 +1,22 @@
 # Learning Ideas
 
-If you have some free time to learn by yourself, or in a pair, here are some ideas to help you use your time fruitfully.
+No matter how much experience, there is always more to learn or practice.
 
-## Code Katas
+Remember, _[practice must be deliberate](./deliberate-practice.md)_.
+
+If you have some free time to learn by yourself, or in a pair, or even a mob, here are some ideas to help you use your time fruitfully.
+
+## I want to...
+
+* [learn how to setup a Ruby development environment](./learn-how-to-setup-ruby.md)
+* [learn a new language](./learn-a-new-language.md)
+* [learn more about software architecture](./learn-software-architecture.md)
+* [practice TDD](./practice-tdd.md)
+* [get feedback on my code style](./get-feedback-on-code.md)
+
+## I'm not sure...
+
```

Comment:
Should there be an entry here? Or future PR?

Reply:
Yeah it's all the topics below... (for now)